components/camel-ahc/src/test/java/org/apache/camel/component/ahc/BaseAhcTest.java

```diff
@@ -88,55 +88,52 @@ protected SSLContextParameters createSSLContextParameters() {
         sslContextParameters.setKeyManagers(kmp);
         sslContextParameters.setTrustManagers(tmp);
         sslContextParameters.setServerParameters(scsp);
-        // use SSLv3 to avoid issue with (eg disable TLS)
-        // Caused by: javax.net.ssl.SSLException: bad record MAC
-        sslContextParameters.setSecureSocketProtocol("SSLv3");
```

Comment:
I think this will break the build on JDK 11, so this still need to be there. @johnpoth can you confirm?

Reply:
The tests pass with JDK 11